test-commands.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+set -e
+
+mkdir testNeuCLI
+cd testNeuCLI
+
+# NEU
+
+commandRun(){
```

Comment:
Can we use another name other than `commandRun` since we are not running a command here? Thanks. Something like `display_cmd`?

Reply:
Oh yea ! Agree.